doc/syntax.md

```diff
@@ -128,29 +451,67 @@ Currenly only spike output is supported.
 
    `process_spikes()`
 
-   concepts for refractoriness
+   emit_spike(): calling this function in the `update`-block results in firing
```

Comment:
I seem to remember that we once discussed replacing "emit_spike()" by just "emit()" as the type of output event can in principle be also something else than spike. However, if the current implementation is like this, we can leave it unchanged until fixed.